wandb/cli/cli.py

```diff
@@ -1104,6 +1114,14 @@ def _parse_settings(settings):
     default=None,
     help="Resume a launch sweep by passing an 8-char sweep id. Queue required",
 )
+@click.option(
+    "--prior_run",
+    "-R",
+    "prior_runs",
+    multiple=True,
+    default=None,
+    help="ID of an existing run to add to this sweep",
```

Comment:
I'm not sure users know what a run id is. I don't want to make the help here too convoluted, but would it help users to note that it is the `public.Run.id` attribute?

Reply:
Run id is referred to on the run documentation. Should be fine.